distribution/docker/build.gradle

```diff
@@ -175,8 +175,10 @@ for (final boolean oss : [false, true]) {
   addBuildDockerImage(oss)
 }
 
-assemble.dependsOn "buildOssDockerImage"
-assemble.dependsOn "buildDockerImage"
+if (System.getProperty("build.docker") != "false") {
```

Comment:
Thanks for looking at this! 

We don't want to bring this behavior back, we would want to remove the property form use and documentation. 

The reason is that since our distribution now includes a docker image, one can't `assemble` without it 
as `assemble` is expected to produce all distributions.